### ISSUE TITLE
Add WASM/web installation docs

### ISSUE THREAD

**Issue by domoritz** [duckdb/duckdb-web#2132]
It would be nice if https://duckdb.org/docs/installation/ listed installation options for wasm/web.

**Comment by szarnyasg** [comment]
@domoritz Sure, we can add it. We'll rolling out a new installation page soon, and will add a Wasm section.

**Comment by domoritz** [comment]
Hmm, the version number is not correct on the installation page. wasm isn't in sync with the duckdb version numbers. https://github.com/duckdb/duckdb-wasm/releases

**Comment by geertjanw** [comment]
Yes, added here https://duckdb.org/docs/installation/ and will investigate your item.
